content/webapp/pages/pages/[pageId].tsx

```diff
@@ -110,7 +110,11 @@ export const getServerSideProps: GetServerSideProps<
   const { pageId } = context.query;
   const siteSection = toMaybeString(context.params?.siteSection);
 
-  if (!looksLikePrismicId(pageId)) {
+  const isLandingPageRenderingAsSubPage =
+    pageId === siteSection &&
+    context.resolvedUrl === `/${siteSection}/${pageId}`;
+
+  if (!looksLikePrismicId(pageId) || isLandingPageRenderingAsSubPage) {
     return { notFound: true };
   }
```

Comment:
@gestchild request for opinion; do you think we want `/visit-us/visit-us` to return a 404, or do we want to redirect to `/visit-us`? I feel like it'd be easy to do but maybe we want to calm it on the redirections..?

Reply:
yeah, I think it is a 404